app/controllers/v1/clusters_controller.rb

```diff
@@ -116,33 +117,48 @@ def self.add_common_params(api)
 
   # GET /projects/:project_id/clusters
   def index
-    authorize @project
+    if @role.nil? and !current_actor.superadmin
```

Comment:
Side note, I've just realised the string distance between superadmin and superman is smaller than I'd've thought.